packages/babel-preset-stage-1/package.json

```diff
@@ -9,6 +9,7 @@
   "main": "lib/index.js",
   "dependencies": {
     "babel-plugin-transform-export-extensions": "^6.22.0",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
```

Comment:
Nit: sort alphabetically for consistency with other files and to avoid overly complicated diffs when someone `npm install --save`s another dependency to this file in the future.